pages/carbonless.vue

```diff
@@ -149,3 +149,12 @@
     </p>
   </section>
 </template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+import { NeoIcon } from '@kodadot1/brick'
+@Component<Carbonless>({
+  components: { NeoIcon },
+})
+export default class Carbonless extends Vue {}
+</script>
```

Comment:
better to create with composition api
```suggestion
<script lang="ts" setup>
import { NeoIcon } from '@kodadot1/brick'
</script>
```